web/src/hash_util.ts

```diff
@@ -129,6 +133,12 @@ export function huddle_with_url(user_ids_string: string): string {
     return "#narrow/dm/" + user_ids_string + "-group";
 }
 
+export function from_conversation_data(conversation_data: ConversationData): string {
```

Comment:
I wasn't quite sure how to name this, since the others are named "by ..." and this is essentially the same as `by_conversation_and_time_url` but from a conversation data object instead of a message.

Reply:
I think `conversation_data_to_url` is probably reasonable? Don't see something great.

Reply:
Or maybe this is the wrong interface, why not just do `by_conversation_and_time_url_by_message_id()`, and accept the ID?